build/tools/install-ports.py

```diff
@@ -104,6 +104,19 @@ def install_binary_packages():
         path = e('/usr/ports/packages/${name}')
         chroot('${WORLD_DESTDIR}', 'env ASSUME_ALWAYS_YES=yes pkg -o DEBUG_LEVEL=3 install -f ${path}', log=logfile)
 
+# XXX
+# This is for DS-System 14.0.0.1. If we don't have a generic mechanism for this by the time there
+# is an update, then this will need to be updated if there is a newer version available.
+# XXX
+def install_asigra():
```

Comment:
This not being a port will force a new fetch every build and may install a corrupted package, as there is no checksum.

Reply:
This will have to do for now. Asigra is very strict and it's security module requires the binaries to have an exact size. If this size changes, then Asigra does not work. Our build process changes the binary sizes and I haven't been able to figure out why as of yet (it's not from stripping, my guess is build machine block size or when the ISO is generated). This is a workaround until this can be sorted and/or we move to new build system.